index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
+          <p>
+            This interface represents a user-defined
+            <a><code>AudioNode</code></a> which lives on the main <a>control
+            thread</a>. The user can create an <code>AudioWorkletNode</code>
+            from an <a><code>AudioContext</code></a> and such a node can be
+            connected with other built-in <a><code>AudioNode</code></a>s to
+            form an audio graph.
+          </p>
+          <dl title="interface AudioWorkletNode : AudioNode" class="idl">
+            <dt>
+              Constructor(BaseAudioContext context, DOMString name, optional
+              AudioWorkletNodeOptions options)
+            </dt>
+            <dd>
+              <p>
+                The construction procedure of an
+                <a><code>AudioWorkletNode</code></a> and the corresponding
+                <a><code>AudioWorkletProcessor</code></a> object consists of
+                several coordinated interactions. The following algorithm
+                specifies the steps in constructing an
+                <a><code>AudioWorkletNode</code></a> with the given parameters.
+                This sequence must be performed by the user agent <a data-lt=
+                "atomic">atomically</a>.
+              </p>
+              <ol>
+                <li>
+                  <em>In the <code>AudioWorkletGlobalScope</code></em>: Locate
+                  a definition of <a><code>AudioWorkletProcessor</code></a>
+                  subclass that is registered under the <code>name</code>
+                  argument. If none is found, an InvalidAccessError exception
+                  MUST be thrown by the <a><code>AudioWorkletNode</code></a>
+                  constructor.
+                </li>
+                <li>Run the <a><code>AudioWorkletProcessor</code></a>
+                constructor, passing a structured clone of the
+                <code>options</code> parameter that was provided to
+                <a><code>AudioWorkletNode</code></a>.
+                  <ol>
+                    <li>Run the super constructor of
+                    <a><code>AudioWorkletProcessor</code> subclass</a>.
+                    </li>
+                    <li>Run the constructor of the
+                    <a><code>AudioWorkletProcessor</code> subclass</a>.
+                    </li>
+                  </ol>
+                </li>
+                <li>
+                  <em>In the <code>window</code> scope</em>: Construct
+                  <a><code>AudioParam</code></a> objects based on the
+                  information from the processor's
+                  <a><code>parameterDescriptors</code></a>.
+                  <ol>
+                    <li>Initialize the value of AudioParam objects with
+                    key-value pairs in the <code>option</code> parameter. Find
+                    the key that matches the name of AudioParam object and
+                    assign the value to it.
+                    </li>
```

Comment:
In terms of non-matching keys that seems good (and important to add).

In terms of non-numeric values: If that is what a native node would do, that seems OK. What happens in the new native property bag constructors if you pass a weird value type?